scripts/decredStakeVoting.md

```diff
@@ -15,13 +15,13 @@ Stake servers stand by 24/7 to ensure votes are counted and rewards are earned,
 
 Ticket holder's votes are essential to the integrity of the Decred network. 
 
-The 5 votes in each new block determine the validity of the previous one. If the previous block is deemed invalid, the transactions and proof-of-work portion of the block reward are stripped from the network. The proof-of-stake portion of of the block reward is always rewarded, whether ticket holders deem the block valid or not. 
+The 5 votes in each new block determine the validity of the previous one. If the previous block is deemed invalid, the transactions and proof-of-work portion of the block reward are stripped from the network. The proof-of-stake portion of the block reward is paid out whether ticket holders deem the previous block valid or not. 
 
-This way, ticket holders are compensated for stopping miner attacks and preventing unwanted chain forks.
+PoS voters are free to reject blocks they seem as invalid and still be rewarded, while PoW miners must comply with the rules set by voters to receive their rewards. This gives PoS voters the power to punish and stop PoW miners misbehavior and prevent unwanted chain forks. 
```

Comment:
First part of this sentence may be redundant, but I think mentioning that miners are punished here is a good addition.